### PR TITLE
Update the "I'm Locked Out" documentation for using a container CLI

### PR DESCRIPTION
**Pull request by jay-em-Ach**
## Proposed change

1. Added steps to change a lost password through the command line when Home Assistant is running in a container.
2. Added a note that the "Change Password" button does not always show up, but switching to another interface will likely show it (I learned that _after_ figuring out this whole `hass` process).

Replacement for #14477 by @hadim and Home Assistant Community feature request "[Reset password from HA Docker Container](https://community.home-assistant.io/t/reset-password-from-ha-docker-container/215537)."

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - [x] I made a change to the existing documentation and used the `current` branch.
  - [ ] I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jay-em-Ach, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18109).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/locked_out.md [hunk 1]
```diff
@@ -40,7 +42,17 @@ Connect a keyboard and monitor to your device.
...
+
+If you are running Home Assistant in a container, you can use the command line in the container with the `hass` command to change your password. The steps below refer to a Home Assistant container in Docker named `ha`. Note that while working in the container, commands will take a few moments to execute.
+  
+1. `docker exec -it ha bash` to open to the container command line
+2. `hass` to create a default user, if this is your first time using the tool
+3. `hass --script auth --config /config change_password existing_user new_password` to change the password
+4. `exit` to exit the container command line
+5. `docker restart ha` to restart the container
+
+### Home Assistant Core
```
Comment (frenck) [inline source/_docs/locked_out.md]: This is not specific to Home Assistant Core installations

**Comment by jay-em-Ach** [comment]
Thanks for the suggestion, @frenck. Corrections are now added.

File: source/_docs/locked_out.md (hunk 1 above)
Comment (jay-em-Ach) [reply source/_docs/locked_out.md]: Thanks for the suggestion, @frenck. Corrections are now added.

**Review by frenck — APPROVED** [review]
Thanks, @jay-em-Ach 👍

**Pull request merged**